pkg/gardenlet/controller/add.go

```diff
@@ -85,6 +88,14 @@ func AddToManager(
 		return fmt.Errorf("failed adding ControllerInstallation controller: %w", err)
 	}
 
+	if err := (&managedseed.Reconciler{
+		Config:          *cfg.Controllers.ManagedSeed,
```

Comment:
You can set this inside `AddToManager` function, since you are anyway passing the whole `cfg`